README.md

```diff
@@ -1 +1,81 @@
-# alpha-finance
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+
+# alpha-finance
+
```

Comment:
The app should be deployed using Heroku, Netlify, or GitHub Pages and a link to the deployed project should be present in the README.